frontend/src/pages/EditRolesPage/index.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { Grid } from '@material-ui/core';
+
+import MemberTable from './components/MemberTable';
+
```

Comment:
in general (well for this codebase), we tried to do all useRequests up at the page level